core/store/src/db.rs

```diff
@@ -193,6 +196,8 @@ lazy_static! {
         col_gc[DBCol::ColBlockOrdinal as usize] = false;
         col_gc[DBCol::ColEpochInfo as usize] = false; // https://github.com/nearprotocol/nearcore/pull/2952
         col_gc[DBCol::ColEpochStart as usize] = false; // https://github.com/nearprotocol/nearcore/pull/2952
+        // TODO: do we need to implement is?
```

Comment:
No you don't need GC. This column should have its own routine to delete data when the contract is deleted.

Reply:
Well, unless the we keep track of the total number of deployed contracts of the same hash we can't delete on contract replacement.

Reply:
> Well, unless the we keep track of the total number of deployed contracts of the same hash we can't delete on contract replacement.

What prevents us from doing that?

Reply:
@olonho For every TODO please add a corresponding tracking issue

Reply:
Guess in this case it was a question to reviewer, and I presume the answer is "no".